pol-core/poltool/PolToolMain.cpp

```diff
@@ -42,34 +43,41 @@ void PolToolMain::showHelp()
               << "            x1 y1 [x2 y2 realm]       writes polmap info to polmap.html\n"
               << "  POLTOOL uncompressgump FileName\n"
               << "        unpacks and prints 0xDD gump from given packet log\n"
-              << "        file needs to contain a single 0xDD packetlog\n";
+              << "        file needs to contain a single 0xDD packetlog\n"
+
+              << "  POLTOOL testfiles [options]\n"
+              << "        Options:\n"
+              << "          outdir=.\n"
+              << "          hsa=0\n"
+              << "          maxtiles=0x3fff\n"
+              << "          width=6144\n"
+              << "          height=4096\n";
 }
 
 int PolToolMain::mapdump()
 {
   short wxl = 5485, wxh = 5500, wyl = 0, wyh = 30;
   const std::vector<std::string>& binArgs = programArgs();
-
   std::string realmname = "britannia";
-  if ( binArgs.size() >= 6 )
+  if ( binArgs.size() >= 7 )
   {
-    realmname = binArgs[5];
+    realmname = binArgs[6];
```

Comment:
Since this changes the parameter list of this command, should we have a note in core-changes?

Reply:
It is broken since mmmh ~3years. Noone ever complained about it. I would silently fix it ;)

Reply:
I'm sure someone will ask "Oh, what is this poltool? Is that new?" ;)